src/Microsoft.AspNetCore.Mvc.Core/Properties/AssemblyInfo.cs

```diff
@@ -4,8 +4,36 @@
 using System.Runtime.CompilerServices;
 using Microsoft.AspNetCore.Mvc.Formatters;
 
-[assembly: InternalsVisibleTo("Microsoft.AspNetCore.Mvc.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
Should leave out these additional IVT changes until and unless we change pubternal classes in Core.

Reply:
Speaking to @rynowak offline, we landed on all of the shared src packages to Core from Abstractions (the ones that Abstractions doesn't use). So we kinda need these now.

Reply:
If Abstractions shares everything with everyone, why do we need additional IVTs in core?

Reply:
Oh, I get it now.